src/NuGet.Core/NuGet.Packaging/ContentModel/ManagedCodeConventions.cs

```diff
@@ -226,10 +224,15 @@ private static NuGetFramework TargetFrameworkName_ParserCore(string name)
             return new NuGetFramework(name, FrameworkConstants.EmptyVersion);
         }
 
-        private static object AllowEmptyFolderParser(string s, PatternTable table)
+        private static object AllowEmptyFolderParser(ReadOnlyMemory<char> s, PatternTable table)
         {
             // Accept "_._" as a pseudo-assembly
-            return PackagingCoreConstants.EmptyFolder.Equals(s, StringComparison.Ordinal) ? s : null;
+            if (MemoryExtensions.Equals(PackagingCoreConstants.EmptyFolder.AsSpan(), s.Span, StringComparison.Ordinal))
+            {
+                return s.ToString();
```

Comment:
Sorry this is a change I hadn't yet pushed to my branch, this should just return `PackagingCoreConstants.EmptyFolder`
```suggestion
                return PackagingCoreConstants.EmptyFolder;
```

Reply:
I have 2 PRs depending on this that will go out today, I'll add it one of those.